ui/src/tool-panel/km-post/dialog/km-post-edit-store.ts

```diff
@@ -62,7 +62,7 @@ const EASTING_MARGIN_BETWEEN_GKS_DEG = 0.01;
 
 // GK-FIN coordinate systems currently only used for the live display of layout coordinates when editing km post
 // positions manually
-export const GK_FIN_COORDINATE_SYSTEMS: [Srid, string][] = [...Array(12)].map(
+export const GK_FIN_COORDINATE_SYSTEMS: [Srid, string][] = [...Array(13)].map(
```

Comment:
GK-kaistat menevät GK19..GK31. `31 - 19 = 12` joten ymmärrän mistä tämä virhe on alunperin johtunut, mutta kyseessä on inklusiivinen range, joten koordinaatistoja on oikeasti 13

Reply:
Vähän meh, että javascriptissä ei ole range-funkkaria tällaisia varten. Sellaisen voisi periaatteessa itse tehdä mutta eipä taida näitä käyttöpaikkojakaan hirveästi olla missä halutaan iteroida joku tietty numeroväli.